src/components/Form.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Form() {
+  return (
+    <div>
+      <form>
+        <input className="title" type="text" placeholder="Title" />
+        <input className="author" type="text" placeholder="Author" />
+        <button className="add-book" type="submit">Add Book</button>
+      </form>
+    </div>
+  );
+}
```

Comment:
[ OPTIONAL] 
- Consider renaming this component from Form to `CreateNewBook` and use it accordingly in the `BookList` component

Reply:
Hi, @t-yanick same with this one, I don't have a `CreateNewBook` component
![image](https://user-images.githubusercontent.com/114549019/213626047-8aa5bb50-62b5-44ab-8b0b-bcb2afbc8022.png)
could you kindly check why there's a discrepancy between our instructions